public void test5() {
    long val1 = 9223372036854775807L;
    int val2 = 2;
    try {
        FieldUtils.safeMultiply(val1, val2);
        fail("Expected ArithmeticException");
    } catch (ArithmeticException ae) {
        assertEquals("Multiplication overflows a long: 9223372036854775807 * 2", ae.getMessage());
    }
}